rstream/client.py

```diff
@@ -31,6 +33,19 @@
 
 logger = logging.getLogger(__name__)
 
+class AddressResolutionMaxRetryError(Exception):
+    pass
+
+class Addr(NamedTuple):
+    host: str
+    port: int
+
+@dataclass
+class AddressResolver:
+    addr: Addr
+    enabled: bool
```

Comment:
Since passing address_resolver is optional itself, I can't see why we could need an additional `enabled: bool` flag

Reply:
@Gsantomaggio can you clarify what the purpose of this is? I used the go/c# implementations as examples

Reply:
@wonesy @qweeze no reason. I thought it could be useful in case you want to bind the property with some script. 
it is not necessary btw.